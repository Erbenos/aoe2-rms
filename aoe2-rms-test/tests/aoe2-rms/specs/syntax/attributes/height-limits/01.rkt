#lang aoe2-rms
; Create a hill and place desert terrain only on the slopes

<ELEVATION-GENERATION>
(create-elevation 7 (base-terrain 'GRASS) (number-of-tiles 3000) (number-of-clumps 1))

<TERRAIN-GENERATION>
(create-terrain 'DESERT
                (base-terrain 'GRASS)
                (land-percent 100)
                (number-of-clumps 9320)
                (height-limits 1 6))
