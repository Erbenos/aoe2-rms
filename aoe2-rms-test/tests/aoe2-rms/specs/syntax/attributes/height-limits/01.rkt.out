<ELEVATION_GENERATION>
create_elevation 7 {
  base_terrain GRASS
  number_of_tiles 3000
  number_of_clumps 1
}
<TERRAIN_GENERATION>
create_terrain DESERT {
  base_terrain GRASS
  land_percent 100
  number_of_clumps 9320
  height_limits 1 6
}
